src/NodaTime/Duration.cs

```diff
@@ -53,11 +54,12 @@ namespace NodaTime
     /// Various operations accept or return a <see cref="Double"/>, in-keeping with durations often being natural lengths
     /// of time which are imprecisely measured anyway. The implementation of these operations should never result in a not-a-number
     /// or infinite value, nor do any operations accept not-a-number or infinite values. Additionally, operations involving
-    /// <c>Double</c> have initially been implemented fairly na�vely; it's possible that future releases will improve the accuracy
+    /// <c>Double</c> have initially been implemented fairly naïvely; it's possible that future releases will improve the accuracy
```

Comment:
It looks like this file was checked into `nodatime/master` with a windows-1252 diacritic symbol, causing that character to break when checked out as UTF-8. 

Assuming that this wasn't intentional, I've replaced the broken character with the UTF-8 version.